server/dive_server/views_override.py

```diff
@@ -30,3 +32,30 @@ def use_private_queue(self, user: dict, privateQueueEnabled: bool):
             constants.UserPrivateQueueEnabledMarker, False
         ),
     }
+
+
+@access.user
+@boundHandler
+@autoDescribeRoute(Description('Get number of outstanding jobs'))
+def countJobs(self):
+    outstanding = (
+        job.Job()
+        .list(
+            user='all',
+            currentUser=self.getCurrentUser(),
+            statuses=[
+                GWJobStatus.QUEUED,
```

Comment:
Sorry it took me a while, was jumping between girder/girder and girder/gider_worker repo's to figure this out.
Looks like it the old method had INACTIVE to QUEUED to Running, but the celery method can skip over QUEUED and have a job go directly from INACTIVE to Running:
https://github.com/girder/girder_worker/blob/master/girder_worker/girder_plugin/status.py#L52

This was initially found based on observation with the fact that it I definitely had 7 jobs lined up and it would only show in the results 1 (which was the running one).  I think for some reason when using the celery method the job stays INACTIVE until it is RUNNING and doesn't have a QUEUED state.
Some this was I could get from quickly going over this as well:
https://github.com/girder/girder_worker/blob/3dab87f738c1a5a440a578c1ea426a83a876e500/HACKING.md

Reply:
Ok, interesting.  I've added `INACTIVE` now.